GVFS/GVFS/CommandLine/SparseVerb.cs

```diff
@@ -435,7 +435,8 @@ private void CheckGitStatus(ITracer tracer, GVFSEnlistment enlistment, HashSet<s
                     this.WriteMessage(tracer, "Either commit your changes or reset and clean");
                 }
 
-                this.ReportErrorAndExit(tracer, SparseVerbName + " was aborted");
+                this.WriteMessage(tracer, string.Empty);

```

Comment:
Should we use `this.Output.WriteLine` instead?  `WriteMessage` will write an empty Info message to the log.